src/main/java/duelistmod/powers/SummonPower.java

```diff
@@ -140,6 +140,10 @@ public void atEndOfTurn(final boolean isPlayer) {
 		ArrayList<DuelistCard> newList = new ArrayList<>();
 		for (DuelistCard c : getCardsSummoned()) {
 			if (!c.hasTag(Tags.SPIRIT)) {
+				if (c.hasTag(Tags.GUSTO) && !c.hasTag(Tags.SPELLCASTER)) {
+					c.customOnTribute(c);
```

Comment:
We'll want to call some more tribute hooks here. If you take a look at the standard tribute() call that most cards use when played, you can see there is a lot of custom logic in there to handle things like updating Giant costs, triggering certain powers that are still hardcoded, etc. I wouldn't be opposed to wrapping up some of this general functionality and placing it into a utility method that can be called here and other places as well, if you're so inclined.